terraform/10-account/iam.terraform-role.tf

```diff
@@ -0,0 +1,16 @@
+module "iam_terraform_role" {
+  source      = "terraform-aws-modules/iam/aws//modules/iam-assumable-role"
+  version     = "5.44.0"
+  create_role = true
+
+  role_name         = "TerraformOperator"
+  role_requires_mfa = false
+
+  custom_role_policy_arns = [
+    "arn:aws:iam::aws:policy/AdministratorAccess",
```

Comment:
We could come back at some point and be explicit about the services we want to give terraform access to. But this will get us moving for now